cli/Cargo.toml

```diff
@@ -122,7 +122,9 @@ hyper-util.workspace = true
 import_map = { version = "=0.20.1", features = ["ext"] }
 indexmap.workspace = true
 jsonc-parser = { workspace = true, features = ["cst", "serde"] }
-jupyter_runtime = { package = "runtimelib", version = "=0.14.0" }
+jupyter_runtime = { package = "runtimelib", version = "=0.19.0", features = [
+  "tokio-runtime",
```

Comment:
`tokio-runtime` is now explicit instead of being the default feature. This has made it easier to support Zed as well since it uses it's own async dispatcher.